src/shared/components/MapBlock.js

```diff
@@ -39,6 +39,9 @@ class MapBlock extends Component {
       <section style={{ width: '100%', height: '500px' }}>
         <h4>coords: {this.props.center}</h4>
         <GoogleMap
+          bootstrapURLKeys={{
+            key: "AIzaSyDjOUM158Gk2APJ7zfGaRMxgVJ5iMs7M-Q"
```

Comment:
There are three reasons why you don't have to worry about that.
1. It's totally free.
2. I can always change the token with just one click on google developer console.
**3. There is a REFERER URL that makes only goober.herokuapp url can request to google map by the key.**

In conclusion, it's totally safe.